Update Readme.md (#1)
DevExpress-Examples/correct-and-incorrect-examples-of-using-height100-in-html: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E4942)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

# Relative height (100%) in DevExpress ASP.NET and DevExtreme controls
<!-- run online -->
**[[Run Online]](https://codecentral.devexpress.com/e4942/)**
<!-- run online end -->

> DevExpress ASP.NET controls do not implement full screen feature (except for [ASPxSplitter](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxSplitter.FullscreenMode)). When a control’s height is set to 100%, it works incorrectly.

Reviewer: - ` do not implement full screen feature` A/the feature? **full-screen feature**? And when you say do not implement, is this the same as support? 
- `it works incorrectly` What does this imply?

Author: ```suggestion
DevExpress ASP.NET controls do not support the full screen feature out of the box (except for [ASPxSplitter](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxSplitter.FullscreenMode)). 

If you set a control’s height to 100%, it will not expand up to the page height, but on the contrary, it will collapse.
```

Reviewer: DevExpress ASP.NET controls do not support the full-screen feature out of the box (except for [ASPxSplitter](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxSplitter.FullscreenMode)). 
If you set a control’s height to 100%, it will collapse instead of expanding to the page height.

Author: ok
```suggestion
DevExpress ASP.NET controls do not support the full-screen feature out of the box (except for [ASPxSplitter](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxSplitter.FullscreenMode)).
If you set a control’s height to 100%, it will collapse instead of expanding to the page height.
```

> The cause of this issue is technological limitation. To set a relative height to an HTML element, **all** its parent HTML elements must have the `height` style initialized. Note that `<html>`, `<body>` and `<form>` tags are also parent containers and their height should also be initialized (directly or in CSS rules).

Reviewer: ```suggestion
This is due to technological limitations. To set a relative height for an HTML element, **all** its parent HTML elements must have the `height` style initialized. Note that `<html>`, `<body>`, and `<form>` tags are also parent containers and their height should also be initialized (directly or in CSS rules).
```
